### ISSUE TITLE
Exception after project reload - cloud-synced project

### ISSUE THREAD

**Issue by cautionbug** [jk1/youtrack-idea-plugin#64]
My project directory is in a cloud-storage directory synced to other computers.
i only use one at a time, and normally the workspace file's changes prompt me to reload the project.
This error occurred several minutes after that reload successfully completed, while the IDE was just sitting idle.

Thank you for the great plugin, by the way!

```
Sorry but parent: Project (Disposed) XXXXXX has already been disposed (see the cause for stacktrace) so the child: null 1360395271: running=false; canceled=false will never be disposed
com.intellij.util.IncorrectOperationException: Sorry but parent: Project (Disposed) XXXXXX has already been disposed (see the cause for stacktrace) so the child: null 1360395271: running=false; canceled=false will never be disposed
	at com.intellij.openapi.util.objectTree.ObjectTree.register(ObjectTree.java:66)
	at com.intellij.openapi.util.Disposer.register(Disposer.java:97)
	at com.intellij.openapi.util.Disposer.register(Disposer.java:93)
	at com.intellij.openapi.progress.util.ProgressWindow.<init>(ProgressWindow.java:125)
	at com.intellij.openapi.progress.util.ProgressWindow.<init>(ProgressWindow.java:84)
	at com.intellij.openapi.progress.impl.BackgroundableProcessIndicator.<init>(BackgroundableProcessIndicator.java:63)
	at com.intellij.openapi.progress.impl.BackgroundableProcessIndicator.<init>(BackgroundableProcessIndicator.java:44)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcessWithProgressAsynchronously(ProgressManagerImpl.java:139)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:341)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:325)
	at com.intellij.openapi.progress.Task.queue(Task.java:115)
	at com.github.jk1.ytplugin.issues.IssueStore.update(IssueStore.kt:21)
	at com.github.jk1.ytplugin.issues.IssueStoreUpdaterComponent$initComponent$1$1.run(IssueStoreUpdaterComponent.kt:28)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.a(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by AndrePek** [comment]
I think this may be the same basic issue as https://github.com/jk1/youtrack-idea-plugin/issues/67, but perhaps in a slightly different context. Certainly, the stacktraces are almost identical, across 30 lines of stacktrace

**Comment by cautionbug** [comment]
Sounds like a reasonable conclusion. i haven't actually experienced this error in a while. Possibly the syncing process was just shaky at some point, i'm not sure.

If you find the cause, fantastic. If i encounter the error again i'll let you know.

Thanks!

**Comment by jk1** [comment]
Fixed in 0.7.1
